test/factories/team_members.rb

```diff
@@ -2,5 +2,20 @@
   factory :team_member do
     name "John Smith"
     role "Board Advisor"
+    group "board"
+    description "Long and wonderful history of employment"
+    image_src "images/john_smith.jpg"
+  end
+
+  trait :board do
```

Comment:
Nice.

Somewhat related to this, the `seeds.rb` file will need updating.  It is pulling its `TeamMember` data from:

https://github.com/OperationCode/operationcode_backend/blob/master/config/team_members.yml

Can you update that .yml file with `group` data, as this will now be a validated attribute.

Reply:
I updated the seeds.rb. Now it calls a new class to seed team members. I updated the rake tasks to use that same class and renamed/added tasks.